app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,5 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```

Comment:
You might want to also add `@import "bootstrap/theme";` then you can delete a bunch of your styles.